Documentation/vpc-backend.md

```diff
@@ -0,0 +1,139 @@
+# Amazon VPC Backend for Flannel
+
```

Comment:
When running in an Amazon VPC we recommend using the aws-vpc backend which, instead of using encapsulation, manipulates IP routes to achieve maximum performance. Because of this, a separate flannel interface is not created.